code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/constants/ServiceDiscoveryConstants.java

```diff
@@ -55,8 +55,12 @@
     public static final String LINK_COMPOSE_CONFIG = "composeConfig";
 
     public static final String LABEL_SERVICE_DEPLOYMENT_UNIT = "io.rancher.service.deployment.unit";
-    public static final String LABEL_PROJECT_NAME = "io.rancher.stack.name";
-    public static final String LABEL_PROJECT_SERVICE_NAME = "io.rancher.stack_service.name";
+    public static final String LABEL_STACK_NAME = "io.rancher.stack.name";
+    public static final String LABEL_STACK_SERVICE_NAME = "io.rancher.stack_service.name";
+    // LEGACY: preserving project_name
+    public static final String LABEL_PROJECT_NAME = "io.rancher.project.name";
+    public static final String LABEL_PROJECT_SERVICE_NAME = "io.rancher.project_service.name";
```

Comment:
Commenting on pre-existing code and also a nit, but, it seems odd to mix dots and underscores here. To me, `io.rancher.project.service.name` seems more natural.

I won't hold up a merge bc of this, just wanted to voice my opinion.